backend/app/Usecases/Tenant/UpdateItemAction.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Usecases\Tenant;
+
+use App\Constants\MessageConst;
+use App\Models\Item;
+use App\Models\Tenant;
+use App\Usecases\Tenant\Exceptions\UploadFileFailedException;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+
+class UpdateItemAction
+{
+    public function __invoke(Tenant $tenant, int $itemID, Item $newItem, ?UploadedFile $newImage): Item
+    {
+        $item = Item::find($itemID);
+
+        if($newImage) {
+            // 古い画像を削除
+            if($item->s3_key) {
```

Comment:
こうっすかね。

```suggestion
            if($item->s3_key !== null) {
```

ref)
https://github.com/ryunosuke121/yumemi-intern-mobile-order-system/pull/4#discussion_r1746535831